packages/amplify_datastore/example/lib/queries_display_widgets.dart

```diff
@@ -1,170 +1,185 @@
-part of sample_app;

```

Comment:
Not sure what this was about - this is the result of adding the license header and running `flutter format`.